### ISSUE TITLE
please fix

### ISSUE THREAD

**Issue by MBYTTE23** [zhuowei/MCPELauncher#1315]

**Comment by MBYTTE23** [comment]
It just won't let me in. I went in to it but then it just was a plain white screen please fix

**Comment by afbmom414** [comment]
Agree. It has to have been close to a month that I've had this issue also. If this isn't going to be fixed, please refund our money.
